s29862-JuliaButkiewicz/Zadania_3/RozwiązanieZadań3.txt

```diff
@@ -41,6 +41,7 @@ Wykonuje powyższe kroki po kolei.
 Zupełnie jak w podpunktach nad tym zadaniem.
 
 WNIOSKI:
-Tym razem wiadomość zapisała się do pliku.
-Przez użycie kombinacji klawiszy dwa razy. 
-
+CTRL + D wysyła tak zwany komunikat EOF - end of file.
+Podczas gdy wcześniej, używając CTRL + C, zabijało to 
+proces tak po wciśnięciu CTRL + D zabija to proces jednak zapisuje to
```

Comment:
^D nic nie zabija. Zakończenie się programu wynika z faktu, że przetworzył on wszystkie dane i nie ma potrzeby by biernie pracował bez nowych danych, które po zamknięciu strumienia wejściowego już nie napłyną.

A zapis do pliku nie jest nawet realizowany przez program `cat` bo ten drukuje wszystko na swoje standardowe wyjście. To `bash` przekierowuje to wyjście do pliku.